x-pack/test/common/services/spaces.ts

```diff
@@ -44,7 +45,10 @@ export function SpacesServiceProvider({ getService }: FtrProviderContext) {
     : undefined;
 
   const axios = Axios.create({
-    headers: { 'kbn-xsrf': 'x-pack/ftr/services/spaces/space' },
+    headers: {
+      'kbn-xsrf': 'x-pack/ftr/services/spaces/space',
+      [ELASTIC_HTTP_VERSION_HEADER]: '2023-10-31',
```

Comment:
Same as above, I had to add the version header to the axios config in order to get the requests to succeed. I can successfully issue the request via curl or postman without the version header, but the axios request will receive status 400 without it.